tests/dummy/app/styles/app.scss

```diff
@@ -23,83 +24,39 @@
   margin-left: 20px;
 }
 
-.dummy-content {
-  flex: 1;
-}
-
-.dummy-header {
-  padding: 20px 20px 10px 0px;
-  font-weight: bold;
-}
-
-.dummy-padding {
-  margin: 10px;
-  padding: 10px 0 10px 20px;
-  display: inline-flex;
-  flex-direction: column;
-}
-
-.dummy-padding-link {
-  margin: 10px;
-  padding: 10px 0 10px 20px;
-}
-
-.dummy-alternate {
-  display: flex;
-  flex-direction: row;
-}
-
-.dummy-text {
-  padding: 5px 5px 5px 30px;
-  width: 100%;
-  font-size: 14px;
-}
+.section {
```

Comment:
Side-note, but we really should make this CSS come from a common package - also, check with @rox163 on what she did for text, we changed the style of the demos a bit